docs/src/components/icons-list.astro

```diff
@@ -2,10 +2,18 @@
 import { Icon } from '@astrojs/starlight/components';
 import { Icons } from '../../../packages/starlight/components/Icons';
 
+interface Props {
+	labels?: {
```

Comment:
This uses the same data structure used in the theme editor for labels which would be nice in case we ever need more labels in the future altho I guess an argument could be made to use a flat string property instead of an object in this case.